jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/AwsExplorerFactory.kt

```diff
@@ -27,13 +25,7 @@ class AwsExplorerFactory : ToolWindowFactory, DumbAware {
         toolWindow.helpId = HelpIds.EXPLORER_WINDOW.id
         if (toolWindow is ToolWindowEx) {
             val actionManager = ActionManager.getInstance()
-            toolWindow.setTitleActions(
-                listOf(
-                    actionManager.getAction("aws.settings.refresh"),
-                    Separator.create(),
-                    FeedbackDialog.getAction(project)
-                )
-            )
+            toolWindow.setTitleActions(listOf(actionManager.getAction("aws.toolkit.explorer.titleBar")))
```

Comment:
Nice clean up

Reply:
also necessary since we had to be able to dynamically add new actions in the `ext-*.xml` plugin extensions.